### PR TITLE
Add clusterSize element to volume xml

### PR DESCRIPTION
**Pull request by meinaLi**
Add clusterSize element to volume xml

Signed-off-by: Meina Li <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type q35 virsh.vol_create.create.positive_test.fs_like_pool.vol_format.v_qcow2.src_pool_type..with_clusterSize
JOB ID     : 6288f486be6b1b12109a1930d7831388a72240c2
JOB LOG    : /root/avocado/job-results/job-2021-07-23T00.03-6288f48/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virsh.vol_create.create.positive_test.fs_like_pool.vol_format.v_qcow2.src_pool_type.dir.non_acl.with_clusterSize: PASS (10.12 s)
 (2/4) type_specific.io-github-autotest-libvirt.virsh.vol_create.create.positive_test.fs_like_pool.vol_format.v_qcow2.src_pool_type.dir.acl_test.with_clusterSize: PASS (23.41 s)
 (3/4) type_specific.io-github-autotest-libvirt.virsh.vol_create.create.positive_test.fs_like_pool.vol_format.v_qcow2.src_pool_type.fs.with_clusterSize: PASS (45.60 s)
 (4/4) type_specific.io-github-autotest-libvirt.virsh.vol_create.create.positive_test.fs_like_pool.vol_format.v_qcow2.src_pool_type.netfs.with_clusterSize: PASS (20.39 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 101.46 s
```

File: virttest/libvirt_xml/vol_xml.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class VolXMLBase(base.LibvirtXMLBase):
 
     __slots__ = ('name', 'key', 'capacity', 'allocation', 'format', 'path',
                  'owner', 'group', 'mode', 'label', 'compat', 'lazy_refcounts',
-                 'encryption', "capacity_unit")
+                 'encryption', 'capacity_unit', 'clusterSize', 'clusterSize_unit')
```
Comment (chunfuwen) [inline virttest/libvirt_xml/vol_xml.py:34]: Add explanation about those 2 newly added parameter after line 30

Comment (meinaLi) [reply virttest/libvirt_xml/vol_xml.py:36]: Sorry for forgetting it and now it has already been added.

**Review by Yingshun — APPROVED** [review]
LGTM

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**